app/js/thisissoon-core/filters/linkDisplay.js

```diff
@@ -9,7 +9,7 @@ angular.module("thisissoon.core").filter("linkDisplay", [
     "$filter",
     function($filter) {
         return function(input) {
-            return input.replace("http://", "").replace("www.", "");
+            return input.replace(/.*?:\/\//g, "").replace(/w{3}./, "").replace(/\/.*/g, "");
         };
```

Comment:
There are tests for this here: https://github.com/thisissoon/thisissoon-frontend/blob/feature/link-filter/tests/unit/thisissoon-core/filters/linkDisplay.js but I could probably add a few more cases. I'll do that now.